scripts/easy-spinnaker/config_hal.sh

```diff
@@ -16,14 +16,13 @@
 export KUBECONFIG=/home/vagrant/.kube/config
 
 # Get nodePort numbers
-DECKNP=$(kubectl get svc spin-deck-np -n oes -o jsonpath='{...nodePort}')
+DECKNP=$(kubectl get svc spin-deck-ui -n oes -o jsonpath='{...nodePort}')
 GATENP=$(kubectl get svc spin-gate-np -n oes -o jsonpath='{...nodePort}')
```

Comment:
Why spin-gate-np? As per https://github.com/OpsMx/enterprise-spinnaker/blob/master/charts/oes/templates/spinnaker-extra/spinsvcs.yaml it should be spin-gate-ui.

Reply:
As discussed, renamed spin-gate-np to spin-gate-ui,  changed the file name as well